src/client/java/teammates/client/scripts/UpdateAccountRequestIndexing.java

```diff
@@ -0,0 +1,81 @@
+package teammates.client.scripts;
+
+import java.util.HashMap;
+import java.util.List;
+
+import jakarta.persistence.TypedQuery;
+import jakarta.persistence.criteria.CriteriaBuilder;
+import jakarta.persistence.criteria.CriteriaQuery;
+import jakarta.persistence.criteria.Root;
+
+import teammates.client.util.BackDoor;
+import teammates.client.util.ClientProperties;
+import teammates.common.datatransfer.SqlDataBundle;
+import teammates.common.exception.HttpRequestFailedException;
+import teammates.common.util.HibernateUtil;
+import teammates.storage.sqlentity.AccountRequest;
+
+/**
+ * Script to trigger indexing for all account requests.
+ */
+public class UpdateAccountRequestIndexing {
+    public static void main(String[] args) {
+        UpdateAccountRequestIndexing updater = new UpdateAccountRequestIndexing();
+        updater.initSession();
+        updater.updateAccountRequestIndexing();
+    }
+
+    /**
+     * Initializes the Hibernate session.
+     */
+    public void initSession() {
+        String connectionUrl = ClientProperties.SCRIPT_API_URL;
+        String username = ClientProperties.SCRIPT_API_NAME;
+        String password = ClientProperties.SCRIPT_API_PASSWORD;
+
+        HibernateUtil.buildSessionFactory(connectionUrl, username, password);
+    }
+
+    /**
+     * Updates the indexing for all account requests using the backdoor and batch processing.
+     */
+    public void updateAccountRequestIndexing() {
+        HibernateUtil.beginTransaction();
+        CriteriaBuilder cb = HibernateUtil.getCriteriaBuilder();
+        CriteriaQuery<AccountRequest> cr = cb.createQuery(AccountRequest.class);
+        Root<AccountRequest> root = cr.from(AccountRequest.class);
+        cr.select(root);
+        TypedQuery<AccountRequest> query = HibernateUtil.createQuery(cr);
+        List<AccountRequest> accountRequests = query.getResultList();
```

Comment:
I have a concern with this. I am not familiar with the rough number of account requests. If there are many of them, it may be necessary to do this with multiple SQL queries using offset and limit, like how you're batching now, just with SQL instead. Though I am not sure about this; if I'm not wrong, there are not that many account requests. Maybe another reviewer can chime in whether this would truly be necessary.

Reply:
the number was mentioned on the Slack thread, not sure if its necessary but I guess a test run with a similar number of rows inserted could be set-up on the staging environment, to see if this poses any issues. In any case I think we should get more opinions on this as well

Reply:
I agree with @jayasting98 that it will be.best if we could do batching for fetching account requests as well, because performance might vary in different laptop a lot. You may refer to the method `doOperation`  in script `PatchUsageStatistics` (reference: #12970 ).